tests/system/action/committee/test_create.py

```diff
@@ -1 +1,137 @@
-# TODO
+from tests.system.action.base import BaseActionTestCase
+
+
+class CommitteeCreateActionTest(BaseActionTestCase):
+    def test_create(self) -> None:
+        self.create_model("organisation/1", {"name": "test_organisation1"})
+        self.create_model("user/20", {"username": "test_user20"})
+        self.create_model("user/21", {"username": "test_user21"})
+        self.create_model("user/22", {"username": "test_user22"})
+        committee_name = "test_committee1"
+        description = "<p>Test Committee</p>"
+
+        response = self.client.post(
+            "/",
+            json=[
+                {
+                    "action": "committee.create",
+                    "data": [
+                        {
+                            "name": committee_name,
+                            "organisation_id": 1,
+                            "description": description,
+                            "member_ids": [20, 21],
+                            "manager_ids": [20, 22],
+                        }
+                    ],
+                }
+            ],
+        )
+        self.assert_status_code(response, 200)
+        model = self.get_model("committee/1")
+        assert model.get("name") == committee_name
```

Comment:
Hello @r-peschke. Thank you for joining this team. I hope you enjoy working and co-working here ... ;-)

In such lines it might be better to use `self.assertEqual(model.get("name"), committee_name)` but you do not have to change it in this closed case.

Reply:
I had the same thought some while ago and talked about this with @reiterl who uses `assert ...` consistently. We came to the conclusion that it really does not make a difference and since the existing tests don't follow a common pattern with this, everyone can do what he wants ;)